threedi_schema/migrations/versions/0228_upgrade_db_1D.py

```diff
@@ -84,10 +84,45 @@
     "pump": ["connection_node_end_id", "zoom_category", "classification"]
 }
 
+ADD_COLUMNS = [
+    ("channel", Column("tags", Text)),
+    ("cross_section_location", Column("tags", Text)),
+    ("culvert", Column("tags", Text)),
+    ("culvert", Column("material_id", Integer)),
+    ("orifice", Column("tags", Text)),
+    ("orifice", Column("material_id", Integer)),
+    # ("orifice", Column("geom", Geometry("LINESTRING"), nullable=False)),
+    # ("pipe", Column("geom", Geometry("LINESTRING"), nullable=False)),
+    ("pipe", Column("tags", Text)),
+    ("pump", Column("tags", Text)),
+    ("weir", Column("tags", Text)),
+    ("weir", Column("material_id", Integer)),
+    # ("weir", Column("geom", Geometry("LINESTRING"), nullable=False)),
```

Comment:
this commented-out geometry column and the two above should be removed

Reply:
removed in 07de80e9be1ce6b14661fe1e87d0eb64c3e58ccc